model/import/Parser/CsvParser.php

```diff
@@ -40,24 +40,32 @@ class CsvParser extends ConfigurableService implements ParserInterface
      */
     public function parseFile(File $file, TemplateInterface $template): ItemImportResult
     {
+        $logger = $this->getLogger();
         $lines = explode(PHP_EOL, $file->readPsrStream()->getContents());
         $header = $this->convertCsvLineToArray($lines[0]);
         $header = $this->trimLine($header);
 
+        $logger->debug('Tabular import: header validation started');
+
         $this->getHeaderValidator()->validate($header, $template);
 
+        $logger->debug('Tabular import: header validation finished');
+
         array_shift($lines);
 
         $items = [];
         $validationReport = [];
         $errorsReport = [];
         $currentLineNumber = 0;
-        $logger = $this->getLogger();
         $lineValidator = $this->getLineValidator();
         $csvLineConverter = $this->getCsvLineConverter();
 
         foreach (array_filter($lines) as $lineNumber => $line) {
             $currentLineNumber = $lineNumber + 1;
+
+            $logger->debug(sprintf('Tabular import: line %s parsing started', $currentLineNumber));
+            $logger->debug(sprintf('Tabular import: line %s raw data is "%s"', $currentLineNumber, $line));
```

Comment:
Why 2 logs?

```suggestion
            $logger->debug(sprintf('Tabular import: parsing line: %s, raw data: "%s"', $currentLineNumber, $line));
```

Reply:
Mb u are right, but there is an AC about started parsing line.
I can combine that in one message.